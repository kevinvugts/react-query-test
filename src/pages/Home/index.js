import React from "react";

const Home = props => {
  return <p>homepage</p>;
};

export default Home;
