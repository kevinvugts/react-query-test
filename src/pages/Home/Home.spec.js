// Link.react.test.js
import React from "react";
import Home from "./index";
import renderer from "react-test-renderer";

describe("Home Page", () => {
  test("Snapshot renders", () => {
    const component = renderer.create(<Home />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
