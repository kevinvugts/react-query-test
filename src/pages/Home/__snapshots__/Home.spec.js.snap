// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page Snapshot renders 1`] = `
<p>
  homepage
</p>
`;
